ext/openssl/openssl.stub.php

```diff
@@ -151,12 +190,16 @@ function openssl_cms_verify(string $input_filename, int $flags = 0, ?string $cer
 /** @param OpenSSLCertificate|array|string $certificate */
 function openssl_cms_encrypt(string $input_filename, string $output_filename, $certificate, ?array $headers, int $flags = 0, int $encoding = OPENSSL_ENCODING_SMIME, int $cipher_algo = OPENSSL_CIPHER_AES_128_CBC): bool {}
 
-/** @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $private_key */
+/**
+ * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string $private_key
+ * @sensitive-param $private_key
+ */
 function openssl_cms_sign(string $input_filename, string $output_filename, OpenSSLCertificate|string $certificate, $private_key, ?array $headers, int $flags = 0, int $encoding = OPENSSL_ENCODING_SMIME, ?string $untrusted_certificates_filename = null): bool {}
 
 /**
  * @param OpenSSLCertificate|string $certificate
  * @param OpenSSLAsymmetricKey|OpenSSLCertificate|array|string|null $private_key
+ * @sensitive-param $private_key
  */
 function openssl_cms_decrypt(string $input_filename, string $output_filename, $certificate, $private_key = null, int $encoding = OPENSSL_ENCODING_SMIME): bool {}
```

Comment:
Same as `openssl_pkcs7_decrypt`, the certificate might be sensitive if it hold PEM for private key and $private_key is not set.

Reply:
Okay, added.